index.html

```diff
@@ -641,26 +641,69 @@ <h2>Accessibility Considerations</h2>
         </p>
     </section>   
 
-    <section id="sec-security-privacy">
-        <h2>Security and Privacy Considerations</h2>
+    <section id="sec-security">
+        <h2>Security Considerations</h2>
+        <p>
+            This specification defines a mechanism to pack, distribute and load MiniApps through a concrete configuration that specifies the app's entry point and different routes (i.e., the manifest's <a href="https://www.w3.org/TR/miniapp-manifest/#pages-member"  data-link-type="dfn">`pages`</a> member). [=User agents=] SHOULD mitigate the potential vulnerability during the initialization and loading phase by limiting the type of resources to be loaded and implementing a sandboxed environment that blocks access to sensitive features by default.  
```

Comment:
What does "sensitive features" mean here? Does it mean something like [powerful features](https://www.w3.org/TR/permissions/#powerful-features)? Or something else?

Reply:
Thanks @xfq for the reference. Indeed, powerful feature is the definition I needed. Changed in e7f771eb701bfa85cee18ea3b673f1e80912e433